src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/JsonSerializerContextTests.cs

```diff
@@ -21,6 +21,27 @@ public static void VariousNestingAndVisibilityLevelsAreSupported()
             Assert.NotNull(NestedPublicContext.NestedProtectedInternalClass.Default);
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public static void ContextMetadataIsImmutable()
```

Comment:
Do already have a test that src-gen `IJsonTypeResolver.GetTypeInfo` implementations are mutable?

Reply:
I'm not sure, at quick glance we general seem to be testing `Combine` scenaria rather than actual mutation. Added.